test/test.js

```diff
@@ -65,23 +58,18 @@ else
   console.assert(revisionInfo.downloaded, `Chromium r${chromiumRevision} is not downloaded. Run 'npm install' and try to re-run tests.`);
 }
 
-// Hack to get the currently-running spec name.
-let specName = null;
-jasmine.getEnv().addReporter({
-  specStarted: result => specName = result.fullName
-});
+const timeout = process.env.DEBUG_TEST || slowMo ?  0 : 10 * 1000;
+
+const {TestRunner, Reporter, Matchers}  = require('../utils/testrunner/');
+const runner = new TestRunner({timeout});
```

Comment:
The main benefit of the test runner is that it runs tests in parallel right? This patch looks fine, but I don't think we should land it until we are sure that the tests run correctly in parallel on the bots.

Reply:
Let's not run tests in parallel.